Tests/Library/Client.tests.ts

```diff
@@ -805,4 +805,10 @@ describe("Library/Client", () => {
             assert.ok(!processorExecuted, "telemetry processor should NOT be executed");
         });
     });
+    describe("#setVersion()", () => {
```

Comment:
The method name should be updated here as well (string and the method call below)